src/screens/FeedScreen/ContentLink.js

```diff
@@ -109,6 +114,19 @@ const ContentLink = ({
   );
 };
 
+ContentLink.propTypes = {
+  item: PropTypes.object,
+  og: PropTypes.object,
+  onPress: PropTypes.func,
+  onHeaderPress: PropTypes.func,
+  onCardContentPress: PropTypes.func,
+  score: PropTypes.number,
+  message: PropTypes.string,
+  messageContainerStyle: PropTypes.object,
+  topics: PropTypes.array,
+  isPostDetail: PropTypes.bool
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes is a great way to enforce type checking in React components. This can help catch bugs early and make the code easier to understand. Good job on this!


<!-- This is an auto-generated comment by OSS CodeRabbit -->